backend/index.js

```diff
@@ -0,0 +1,14 @@
+import express from 'express'
+
+const PORT  = 5000;
+
+
+const app = express()
+
+app.get('/*', (req, res) =>{
+    
+    res.status(200).json('Hello world!')
+
+})
+
+app.listen(PORT, () => console.log("SERVER STARTED ON PORT " +PORT ))
```

Comment:
@stas3729 Add newline in the end of the file

Reply:
Here how good code style looks like
`console.log("SERVER STARTED ON PORT " + PORT)`